index.js

```diff
@@ -273,3 +273,33 @@ form.addEventListener('submit', (e) => {
     form.submit();
   }
 });
+
+// local storage
+function saveUserDetails() {
+  const formData = new FormData(form);
+  const fullname = formData.get('fullname');
+  const email = formData.get('email');
+  const feedback = formData.get('feedback');
+  const myFormData = { name: fullname, email, feedback };
+  localStorage.setItem('myFormData', JSON.stringify(myFormData));
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const emailValid = validateEmail(emailInput, INPUT_LOWERCASE);
+  if (emailValid) {
+    form.submit();
+    saveUserDetails();
+  }
+});
+
+const stored = localStorage.getItem('myFormData');
+
+function getUserDetails(localObj) {
+  const userDetails = JSON.parse(localObj);
+  form.fullname.value = userDetails.name;
+  form.email.value = userDetails.email;
+  form.feedback.value = userDetails.feedback;
+}
```

Comment:
- [ ] [OPTIONAL] It is better to check out if the data is saved in the browser or not as this may cause an error if the `stored` variable is null.

```javascript
function getUserDetails(localObj) {
  if(localObj === null) {
    return;
  }
  const userDetails = JSON.parse(localObj);
  form.fullname.value = userDetails.name;
  form.email.value = userDetails.email;
  form.feedback.value = userDetails.feedback;
}
```